test/TestServerProjectsLowLevel/dpg-customization/Generated/DPGClient.cs

```diff
@@ -318,12 +323,17 @@ public virtual Operation<BinaryData> Lro(WaitUntil waitUntil, string mode, Reque
         {
             Argument.AssertNotNull(mode, nameof(mode));
 
-            using var scope = ClientDiagnostics.CreateScope("DPGClient.Lro");
+            return LroImplementation("DPGClient.Lro", waitUntil, mode, context);
+        }
+
+        private Operation<BinaryData> LroImplementation(string diagnosticsScopeName, WaitUntil waitUntil, string mode, RequestContext context)
```

Comment:
Why did we choose to use this approach instead of using the suppress nested span functionality Liudmila provided us with?

Reply:
As we have discussed last Wednesday, we can't suppress nested scope when `UpdateStatusAsync` is called for the operation created by grow-up method that uses different scope name than the protocol one. This is similar to pageable case. But maybe I don't understand how suppression works. @ShivangiReja , @lmolkova - can you please comment on this one?

Reply:
For paging we added private methods in order to fix https://github.com/Azure/azure-sdk-for-net/issues/28124 issue
>the protocol pageable method creates a span using the protocol method name, and there isn't a way to wrap the Get/Next Page network call in a span that uses the name of the customer-invoked convenience method.

Also, I was getting [this](https://github.com/azure/autorest.csharp/issues/2075#issuecomment-1069757581) issue when running [this](https://github.com/ShivangiReja/azure-sdk-for-net/blob/MetricsAdvisor-Experiment/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/MetricFeedbackLiveTests.cs#L203) testcase without [ForwardClientCalls] on the grow-up method.